bootstrap/shoppingmall_review_seed.py

```
@@ -209,89 +234,68 @@ def generate_all_reviews(count: int = 1000) -> list[dict]:
-                    "created_at": r["created_at"],
-                })
+            for review in batch:
+                values.append(
+                    {
+                        "product_id": review["product_id"],
+                        "user_id": review["user_id"],
+                        "rating": review["rating"],
+                        "content": review["content"],
+                        "created_at": review["created_at"],
+                    }
+                )
```

Ruff (0.15.10)

[warning] Use a list comprehension to create a transformed list

(PERF401)